lib/y2lan_restart_common.pm

```diff
@@ -141,11 +150,13 @@ sub check_network_status {
     else {
         assert_script_run 'dig suse.com|grep \'status: NOERROR\'';    # test if conection and DNS is working
     }
-    assert_script_run 'cat journal.log';                              # print journal.log
-    if ($expected_status eq 'restart') {
-        assert_script_run '[ -s journal.log ]';                       # journal.log size is greater than zero (network restarted)
+    assert_script_run 'cat journal.log';
+    if (script_run('[ -s journal.log ]')) {
+        assert_equals($expected_status, 'no_restart', "Network restart didn't occur, even though it was expected.");
```

Comment:
A comment would help here :)

Reply:
Comment added.

Reply:
Also it would be more readable if you convert the result of size to its real meaning, so if the file has content, means 'restart', so then you compare `$expected_status` with `$actual_status`, otherwise it looks like you compare with your own expectations here, but it is fine as it is, just a little cumbersome, it is just a suggestion.